### DISCUSSION TITLE
2.1 / 2.0 Running on localhost only showing register page

### DISCUSSION THREAD

**Discussion by Mattyy94** [plausible/analytics#4160] [Self-Hosted Support] [answered]
Hi everyone,

I am testing plausible, but before I add it to a VPS I want to test it on my local machine. I have everything setup without reverse-proxy. and plausible is accessible thought localhost:8000

![image](https://github.com/plausible/analytics/assets/8201391/9d43e58f-c1a5-460e-ac65-8acfc4ecde96)
The register page is showing up nicely, but I can't go to the login page, since I've setup admin credentials in the pausible-conf so I can login. And play around. 

But I can't go to the login page it will just reload the register page. Any Ideas how i can solve this?

**Comment by ruslandoga** [comment]
👋 @Mattyy94 

> I've setup admin credentials in the pausible-conf 

These have been deprecated for some time and are ignored now.

> But I can't go to the login page it will just reload the register page. Any Ideas how i can solve this?

The registration page would be shown until at least one user exists in the database. So the solution is to register :)

**Reply by ruslandoga** [reply]
Just in case, here's the official starting guide: https://github.com/plausible/community-edition#quick-start

**Reply by Mattyy94** [reply]
Yes I understand this but registering is also not working, seems like that the database connection is not working propperly?

When i press the create account button the following error is shown in the console.
![image](https://github.com/plausible/analytics/assets/8201391/bafc9d62-0110-437b-a271-e8e223fe6ecb)

**Reply by ruslandoga** [reply]
I see. Are there any errors in `docker compose logs`? Can you please show more of the screenshot / browser logs, the `_csrf_token` part looks interesting.

**Reply by Mattyy94** [reply]
The only error I see there is related to a orgin check that cannot be done on localhost:

![image](https://github.com/plausible/analytics/assets/8201391/78e97240-be7e-4010-9b58-63c9f6e4bd39)


`_csrf_token error:`
![image](https://github.com/plausible/analytics/assets/8201391/1adc19a6-7ab6-434f-80c5-3af3effcb1e6)
`WebSocket connection to 'ws://127.0.0.1:8000/live/websocket?_csrf_token=ACYjBVgYFTEjDFxgN0BeBDEWGysjDxtf8MHhlZlfIHhYSssopejdJK6r&_mounts=0&_live_referer=undefined&vsn=2.0.0' failed:`

**Reply by ruslandoga** [reply]
What is your `BASE_URL` set to in `plausible-conf.env`?

**Reply by Mattyy94** [reply]
That is set to: `BASE_URL=http://localhost:8000`

**Reply by ruslandoga** [reply, marked as answer]
Then you should be accessing Plausible via http://localhost:8000 and not http://127.0.0.1:8000 :)

**Reply by Mattyy94** [reply]
Wow oke this is working! Thanks for your help! I think I opent the link though Docker... which went to `127`, it is now working!
